pkgs/by-name/wi/wivrn/package.nix

```diff
@@ -36,58 +38,33 @@
 , vulkan-loader
 , vulkan-tools
 , x264
+, cudaSupport ? config.cudaSupport
 }:
-stdenv.mkDerivation (finalAttrs: {
-  pname = "wivrn";
-  version = "0.15";
-
-  src = fetchFromGitHub {
+let
+  wivrnVersion = "0.15";
+  wivrnSrc = fetchFromGitHub {
     owner = "meumeu";
     repo = "wivrn";
-    rev = "v${finalAttrs.version}";
+    rev = "v${wivrnVersion}";
     hash = "sha256-RVRbL9hqy9pMKjvzwaP+9HGEfdpAhmlnnvqZsEGxlCw=";
   };
+  monadoVersion = builtins.head (builtins.elemAt (builtins.split
+    "monado\n +GIT_TAG +([A-Za-z0-9]+)"
+    (builtins.readFile (wivrnSrc + "/CMakeLists.txt"))
+  ) 1);
```

Comment:
I'll have to let the other reviewers decide on this.

Reply:
This is necessary for the automatic update functionality to work.